Activities/Module13 Activity.md

```diff
@@ -146,3 +146,217 @@ In case your server is is not responding to requests after a long period of inac
 14. Click on the "Environment" tab.
 15. Add a new environment variable called `CLIENT_URL`. For the value, add the client URL (make sure you are adding this env. variable in the server's settings, not the client's). You should now have two environment variables for your server: `MONGDB_URI` and `CLIENT_URL`.
 16. Visit the client URL in your browser to run the application.
+
+
+## Alternative Steps to Set Up Heroku Deployment for Your Backend Service
+
+## Step 1: Enroll in GitHub Student Developer Pack and Heroku for Students
```

Comment:
Make sub-headings these H3s (in the bottom ones as well) - main heading is H2
`###` instead of `##`

Reply:
Done